be/src/olap/rowset/segment_v2/column_zone_map.h

```diff
@@ -73,7 +73,8 @@ class ColumnZoneMapBuilder {
     std::unique_ptr<Field> _field;
     // memory will be managed by arena
     ZoneMap _zone_map;
-    char* _max_string_value;
+    char* _max_char_value;
+    char* _max_varchar_value;
```

Comment:
`_max_char_value` and `_max_varchar_value` are not used, should be removed

Reply:
👌